src/common/components/paymentMethods/editPaymentMethod/editPaymentMethod.modal.component.js

```diff
@@ -0,0 +1,57 @@
+import angular from 'angular';
+
+import template from './editPaymentMethod.modal.tpl';
+
+import paymentMethodDisplay from 'common/components/paymentMethods/paymentMethodDisplay.component';
+
+import loadingOverlay from 'common/components/loadingOverlay/loadingOverlay.component.js';
+
+import bankAccountForm from '../bankAccountForm/bankAccountForm.component';
+import creditCardForm from '../creditCardForm/creditCardForm.component';
+
+let componentName = 'editPaymentMethodModal';
+
+class editPaymentMethodModalController{
```

Comment:
How hard do you think it would be to use https://github.com/CruGlobal/give-web/blob/master/src/common/components/paymentMethods/addNewPaymentMethod/addNewPaymentMethod.component.js for this? We could add that alert at the top that shows the current card details. We could rename it to addUpdatePaymentMethod.component.js or something. That might be too long... You could leave it for now though and we can get to it later. I'm going to need this or the combined component for the edit recurring gifts modal.